cli/cmd/runtime/start.go

```diff
@@ -56,6 +57,18 @@ type Config struct {
 	AllowHostAccess bool `default:"false" split_words:"true"`
 	// Redis server address host:port
 	RedisURL string `default:"" split_words:"true"`
+	// Usage sink type: noop, console, kafka, gcs
+	UsageSinkType string `default:"console" split_words:"true"`
+	// Usage sink period in millis
+	UsageSinkPeriod int `default:"1000" split_words:"true"`
+	// Max queue size of a usage client
+	UsageMaxQueueSize int `default:"1000" split_words:"true"`
+	// Kafka brokers of a usage sink
+	UsageKafkaBrokers string `default:"" split_words:"true"`
+	// Kafka topic of a usage sink
+	UsageKafkaTopic string `default:"" split_words:"true"`
+	// GCS bucket of a usage sink
+	UsageGCSBucket string `default:"" split_words:"true"`
```

Comment:
These emitters can be used to emit metrics that are not usage related as well, like health events. How about renaming to generic Emitters ?

Reply:
Renamed to Activity